_maps/shuttles/nanotrasen/nanotrasen_gecko.dmm

```diff
@@ -1548,6 +1548,7 @@
 /obj/item/clothing/glasses/meson/engine/tray,
 /obj/item/tank/internals/emergency_oxygen/engi,
 /obj/item/radio/intercom/directional/north,
+/obj/item/gun/energy/plasmacutter,
```

Comment:
Shouldn't those be in salvage tech lockers instead? Salvaging's in the name!

Reply:
my brain told me engineering tool so into the engi locker, but yeah this makes sense. It's been moved.